<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportWidget</class>
 <widget class="QWidget" name="ReportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ROOMID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_roomid">
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Switch Times:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_times">
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>start_time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>end_time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>velocity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>start_temp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>end_temp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>fee</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
